core/variant/variant_op.cpp

```diff
@@ -233,6 +285,58 @@ class OperatorEvaluatorModNZ {
 	static Variant::Type get_return_type() { return GetTypeInfo<R>::VARIANT_TYPE; }
 };
 
+template <>
+class OperatorEvaluatorModNZ<Vector2i, Vector2i, Vector2i> {
+public:
+	static void evaluate(const Variant &p_left, const Variant &p_right, Variant *r_ret, bool &r_valid) {
+		const Vector2i &a = *VariantGetInternalPtr<Vector2i>::get_ptr(&p_left);
+		const Vector2i &b = *VariantGetInternalPtr<Vector2i>::get_ptr(&p_right);
+		if (unlikely(b.x == 0 || b.y == 0)) {
+			r_valid = false;
+			*r_ret = "Module by zero error";
```

Comment:
Not sure "Module" is the best word here, perhaps "Modulus".

However, the `%` operator in C++ is not actually a modulus operator, it is a remainder operator. The difference is with negative numbers, `-1 % 3` is `-1` in C++, but would be `2` with modulus. So maybe instead this should be "Remainder" here, and `OperatorEvaluatorModNZ` could be `OperatorEvaluatorRemNZ`.